scripts/update/Updater.php

```diff
@@ -127,5 +136,31 @@ public function update($initialVersion)
 
             $this->setVersion('3.9.0');
         }
+
+        if ($this->isVersion('3.9.0')) {
```

Comment:
don"t need to bump two time the version

Reply:
The manfiest bump is missing

Reply:
Adjusting, btw the manifest was not missing